src/AsyncStorageLogger.ts

```diff
@@ -0,0 +1,314 @@
+import CONFIG from './config/config'
```

Comment:
I see a lot of code duplicates with StorageLogger.ts class.
Let's create as many helper functions in StorageLogger as we can, for them to do such things:
![image](https://user-images.githubusercontent.com/23074851/179177305-94487834-a6e8-4218-8caa-e18685b91f0a.png)

Then let's try to reuse the StorageLogger by overloading its functions in AsyncStorageLogger where we need to